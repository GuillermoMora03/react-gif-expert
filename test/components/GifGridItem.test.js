import { render } from "@testing-library/react";
import { GifGridItem } from "../../src/components/GifGridItem";

describe('pruebas en GifGridItem', () => { 

    const title = 'Saitama';
    const url = 'https://on-punch.com/saitama/jpg';
    
    test('debe de hacer match con el snapshot', () => { 

        const { container } = render( <GifGridItem title={ title } url={ url }/> );
        expect( container ).toMatchSnapshot();

     })

 })