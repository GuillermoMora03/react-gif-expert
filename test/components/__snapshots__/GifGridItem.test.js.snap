// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`pruebas en GifGridItem debe de hacer match con el snapshot 1`] = `
<div>
  <div
    class="card"
  >
    <img
      alt="Saitama"
      src="https://on-punch.com/saitama/jpg"
    />
    <p>
      Saitama
    </p>
  </div>
</div>
`;
